x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_form.scss

```diff
@@ -0,0 +1,9 @@
+.actActionForm__disabledActionWarningTitle {
+  color: #BD271E;
+  font-size: 14px;
+  margin-left: 10px;
+}
+
+.actActionForm__getMoreActionsLink {
+  float: right;
```

Comment:
Try not to use a float. @andreadelrio. Might want to convert whatever this is to using a flex group.